documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -281,7 +281,7 @@ The API invoker has to provide in the authorization request (/bc_authorize) a lo
 
     ```
     POST /bc-authorize HTTP/1.1
-    Authorization: Basic {Credentials}
+    Authorization: ...
```

Comment:
Actually, we can remove the authorization header entirely and just leave the example without explicitly describing client authentication, since this example is really meant for login_hint. I say this because the Authorization header would not apply to private_key_jwt, for example.